website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -55,49 +59,63 @@ Provide your new app access to Azure DevOps:
 
 ## Add another redirect URI
 
-You also need to add another redirect URI to your Azure AD application. This redirect URI will be used to authenticate the service user for headless actions in deployment environments.
+An Entra ID admin needs to add another redirect URI to your Entra ID application. This redirect URI will be used to authenticate the service user for headless actions in deployment environments.
 
-1. Navigate to your Azure AD application.
+1. Navigate to your Entra ID application.
 2. Select the link next to **Redirect URIs**
 3. Click **Add URI** and add the URI, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan:
 `https://YOUR_ACCESS_URL/complete/azure_active_directory_service_user`
 4. Click **Save**.
 
 <Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/redirect-uri.gif" title="Adding the Service User redirect URI"/>
 
+## Create a client secret
 
+An Entra ID admin needs to complete the following steps:
 
+1. Navigate to your Entra ID application.
+2. Select **Certificates and Secrets** from the left navigation panel.
+3. Select **Client secrets** and click **New client secret**
+4. Give the secret a description and select the expiration time. Click **Add**.
+5. Copy the **Value** field and securely share it with the dbt Cloud account admin who will complete the setup. 
 
 ## Connect Azure DevOps to your new app
-If you have already connected your Azure DevOps account to Active Directory, then you can proceed to [Connecting a service user](#connecting-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Active Directory App you just created:
+
+An Azure admin will need one of the following permissions in both the Entra ID and Azure DevOps environments:
```

Comment:
Is "Azure admin" another persona? Or is ti the same as one of the others?

If it is not, can we add it at the top in the list of personas a(and maybe mention what sort of admin and/or access it requires)